docs/rest/src/files/CheckFileInfo.rst

```diff
@@ -212,6 +212,12 @@ WOPI implementation meets the requirements for a particular property.
 
         ..  versionadded:: 2015.08.03
 
+    SupportsSessionRefreshInfo
+        A **Boolean** value that indicates that the host supports the following WOPI operations:
```

Comment:
These aren't WOPI operations. They're PostMessages. So this should be worded to indicate that if true, the Host *must* support PostMessage - so they have to set PostMessageOrigin properly, etc.